### PR TITLE
Fix react bootstrap typedef

### PR DESCRIPTION
**Pull request by subash-a**
Fixing the react bootstrap type definitions for Panel and PanelProps

### CODE REVIEW

File: react-bootstrap/react-bootstrap.d.ts [hunk 1]
```diff
@@ -128,7 +128,7 @@ declare module "react-bootstrap" {
     interface MenuItemClass extends React.ComponentClass<MenuItemProps> { }
     var MenuItem: MenuItemClass;
 
-
+	type renderable = string | number | Element;
```
Comment (myitcv) [inline react-bootstrap/react-bootstrap.d.ts]: Is this really the intended type?

Comment (subash-a) [inline react-bootstrap/react-bootstrap.d.ts]: Yes Paul, based on the description [here](https://facebook.github.io/react/docs/reusable-components.html) the explanation says that anything which is of type _node_ will be 

```
 // Anything that can be rendered: numbers, strings, elements or an array
 // (or fragment) containing these types.
 optionalNode: React.PropTypes.node,
```

which is the type I have tried to create

Comment (subash-a) [inline react-bootstrap/react-bootstrap.d.ts]: @myitcv This has now been changed to one of the react based types `ReactNode` which covers all the types mentioned in the document.

**Comment by myitcv** [comment]
:+1:

**Pull request merged**